projects/v2/src/assets/omap-sheet-config.json

```diff
@@ -568,7 +750,257 @@
+        "xlsx": "https://cdn.humanatlas.io/digital-objects/omap/14-kidney-cycif/v1.0/assets/omap-14-kidney-cycif.xlsx",
+        "as": 8,
+        "ct": 9,
+        "bp": 17,
+        "sheetId": "",
+        "gid": "",
+        "url": "https://purl.humanatlas.io/omap/14-kidney-cycif/v1.0"
       }
     ]
+},
+{
+"omapId": "OMAP-15",
+"name": "Intestines",
+"display": "Intestines",
+"tissuePreservationMethod": "FFPE",
+"imagingMethod": "CODEX",
+"config": {
+  "bimodal_distance_x": 250,
+  "bimodal_distance_y": 60,
+  "width": 700,
+  "height": 2250
+},
+"uberon_id": "UBERON:0000160",
+"version": [
+  {
+    "viewValue": "v1.0",
+    "hraVersion": "v2.1",
+    "value": "OMAP-15-Intestines-v1.0",
+    "csvUrl": "https://cdn.humanatlas.io/digital-objects/omap/15-intestine-codex/v1.0/assets/omap-15-intestine-codex.csv",
+    "xlsx": "https://cdn.humanatlas.io/digital-objects/omap/15-intestine-codex/v1.0/assets/omap-15-intestine-codex.xlsx",
+    "as": 30,
+    "ct": 38,
+    "bp": 54,
+    "sheetId": "",
+    "gid": "",
+    "url": "https://purl.humanatlas.io/omap/15-intestine-codex/v1.0"
+  }
+]
+},
+{
+"omapId": "OMAP-16",
+"name": "Lung",
+"display": "Lung",
+"tissuePreservationMethod": "FFPE",
+"imagingMethod": "CODEX",
+"config": {
+  "bimodal_distance_x": 250,
+  "bimodal_distance_y": 60,
+  "width": 700,
+  "height": 2250
+},
+"uberon_id": "UBERON:0002048",
+"version": [
+  {
+    "viewValue": "v1.0",
+    "hraVersion": "v2.1",
+    "value": "OMAP-16-Lung-v1.0",
+    "csvUrl": "https://cdn.humanatlas.io/digital-objects/omap/16-lung-codex/v1.0/assets/omap-16-lung-cell-codex.csv",
+    "xlsx": "https://cdn.humanatlas.io/digital-objects/omap/16-lung-codex/v1.0/assets/omap-16-lung-cell-codex.xlsx",
+    "as": 3,
+    "ct": 35,
+    "bp": 34,
+    "url": "https://purl.humanatlas.io/omap/16-lung-codex/v1.0",
+    "sheetId": "",
+    "gid": ""
+  }
+]
+},
+{
+"omapId": "OMAP-17",
+"name": "Thymus",
+"display": "Thymus",
+"tissuePreservationMethod": "1% PFA Fixed Frozen",
+"imagingMethod": "IBEX",
+"config": {
+  "bimodal_distance_x": 250,
+  "bimodal_distance_y": 60,
+  "width": 700,
+  "height": 2250
+},
+"uberon_id": "UBERON:0002370",
+"version": [
+  {
+    "viewValue": "v1.0",
+    "hraVersion": "v2.1",
+    "value": "OMAP-17-Thymus-v1.0",
+    "csvUrl": "https://cdn.humanatlas.io/digital-objects/omap/17-thymus-ibex/v1.0/assets/omap-17-thymus-ibex.csv",
+    "xlsx": "https://cdn.humanatlas.io/digital-objects/omap/17-thymus-ibex/v1.0/assets/omap-17-thymus-ibex.xlsx",
+    "as": 7,
+    "ct": 30,
+    "bp": 43,
+    "url": "https://purl.humanatlas.io/omap/17-thymus-ibex/v1.0",
+    "sheetId": "",
+    "gid": ""
+  }
+]
+},
+{
+"omapId": "OMAP-18",
+"name": "Lymph Node",
+"display": "Lymph Node",
+"tissuePreservationMethod": "FFPE",
+"imagingMethod": "Cell DIVE-IBEX",
+"config": {
+  "bimodal_distance_x": 250,
+  "bimodal_distance_y": 60,
+  "width": 700,
+  "height": 2250
+},
+"uberon_id": "UBERON: 0000029",
```

Comment:
Fix malformed uberon ids with a space in it.. `UBERON: 0000029` => `UBERON:0000029`

Reply:
found 2 instances, both with lymph node; corrected. pushed